packages/editor/editor.css

```diff
@@ -969,13 +969,17 @@ input,
 
 .tl-rich-text p {
 	margin: 0;
+	/* Depending on the extensions, <p> tags can be empty, without a <br />. */
+	min-height: 1lh;
 }
 
 .tl-rich-text ul,
 .tl-rich-text ol {
 	text-align: left;
 	margin: 0;
 	padding-left: 3.25ch;
+	/* Some resets, like Tailwind, nix the list styling. */
+	list-style: revert;
```

Comment:
Are there any other styles that we rely on, but might be reset?

Reply:
as far as i know, i _think_ this will be sufficient. 🤞